README.md

```diff
@@ -152,6 +152,7 @@ The component accepts the following props:
 |**`customRowRender `**|function||Override default row rendering with custom function. `customRowRender(data, dataIndex, rowIndex) => React Component`
 |**`customSort`**|function||Override default sorting with custom function. `function(data: array, colIndex: number, order: string) => array`
 |**`customSearch `**|function||Override default search with custom function. `customSearch(searchQuery: string, currentRow: array, columns: array) => boolean`
+|**`customTableSearch `**|function||Render a custom table search. `customTableSearch(searchText: string, handleSearch, hideSearch, options) => React Component`
```

Comment:
To keep the API somewhat consistent, lets call this option `customSearchRender`.